client/blocks/sideload-a8c-analytics-banners/index.tsx

```diff
@@ -0,0 +1,105 @@
+			{ args: { cookieUrl: 'https://automattic.com/cookies/' } }
+		),
+		v2AdvertisingOptionHeading: translate( 'Advertising' ),
+		v2AdvertisingOptionText: translate(
+			'These cookies are set by us and our advertising partners to provide you with relevant content and to understand that content’s effectiveness. They may be used to collect information about your online activities over time and across different websites to predict your preferences and to display more relevant advertisements to you. These cookies also allow a profile to be built about you and your interests, and enable personalized ads to be shown to you based on your profile.'
+		),
+		v1ButtonText: translate( 'Got It!' ),
+		v2CustomizeButtonText: translate( 'Customize' ),
+		v2AcceptAllButtonText: translate( 'Accept All' ),
+		v2AcceptSelectionButtonText: translate( 'Accept Selection' ),
+	},
+};
+
+const onA8cAnalyticsLoaded = () => {
+	debug( '[ a8c-analytics.js ] Script has been loaded, Cookie Banner may be displayed' );
+};
+const onA8cTrackersFired = () => {
+	debug( '[ a8c-analytics.js ] Triggering allowed trackers in regard to user consent' );
+	loadTrackingScripts();
+};
+
+/**
+ * Component that loads a8c-analytics.js scripts into the React app functionality.
+ * That includes:
+ * - Cookie Banner
+ * - Do Not Sell dialog
+ *
+ * The script automatically identifies if user should see the banner, for reference see:
+ * fbhepr%2Fkers%2Fjcpbz%2Fjc-pbagrag%2Fzh-cyhtvaf%2Fn8p-nanylgvpf%2Fn8p-nanylgvpf.wf%3Fe%3D25o9qr55-og
+ */
+const SideloadAnalyticsBanners = () => {
+	useEffect( () => {
+		// Assign config that will be used by a8c-analytics.js script
+		window.a8cAnalyticsConfig = a8cAnalyticsConfig;
+		// Add listener that will get triggered once a8c-analytics.js will be successfully loaded
+		document.addEventListener( 'a8c-analytics:loaded', onA8cAnalyticsLoaded );
+		document.addEventListener( 'a8c-analytics:fire-sensitive-pixels', onA8cTrackersFired );
+		// Load the script into the html document
+		loadScript(
+			`//s1.wp.com/wp-content/mu-plugins/a8c-analytics/a8c-analytics.js?v=${ A8C_ANALYTICS_CACHE_VERSION }`,
+			( error?: { src: string } ) => {
+				if ( error ) {
+					debug(
+						"[ a8c-analytics.js ] Couldn't load the script! Aborting displaying Cookie Banner"
+					);
+				}
+			}
+		);
+
+		return () => {
+			document.removeEventListener( 'a8c-analytics:loaded', onA8cAnalyticsLoaded );
+			document.removeEventListener( 'a8c-analytics:fire-sensitive-pixels', onA8cTrackersFired );
+		};
+	}, [] );
+
+	return null;
+};
+
+export default SideloadAnalyticsBanners;
```

Comment:
A few naming nits on the file name and component name: I find `Sideload` redundant, since it's indicative by how we load it (async or not). The `a8c` part can also feel redundant - although maybe it's not and it provides a reference to the particular analytics project? Just food for thought and a nudge in case this naming be improved with clarity and transparency in mind.